docs/conf.py

```diff
@@ -351,11 +351,6 @@
     "grpc": ("https://grpc.github.io/grpc/python/", None),
     "proto-plus": ("https://proto-plus-python.readthedocs.io/en/latest/", None),
     "protobuf": ("https://googleapis.dev/python/protobuf/latest/", None),
-    "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
-    "pydata-google-auth": (
-        "https://pydata-google-auth.readthedocs.io/en/latest/",
-        None,
-    ),
```

Comment:
Do we really mean to drop these mappings?

Reply:
Good catch. No, we still want these.